### DISCUSSION TITLE
SO 2.4.10 help please (Install Error and no Suricata Alerts)

### DISCUSSION THREAD

**Discussion by plaotec** [Security-Onion-Solutions/securityonion#11200] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

Got Install had a problem at the end of the install on ESXi v7.0 .. so-status shows everything GUI ... GUI dashboards, kibana all ok, can see live connections in kibana ... but not seeing any suricata alerts... tried editing the /opt/so/conf/suricata.yaml file, to change the homenet, but after change, and restarted so-suricata, the homenet went back to the default values.  sosetup.log file attached .. thanks

[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/12468343/sosetup.log)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by plaotec** [comment]
![1](https://github.com/Security-Onion-Solutions/securityonion/assets/63425694/128f793f-6a3c-4002-8be9-06b1c695e6f3)
![2](https://github.com/Security-Onion-Solutions/securityonion/assets/63425694/4467b3ec-83a4-49cb-a64f-aa7bac9063db)

**Comment by plaotec** [comment]
[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/12468375/sosetup.log)

**Comment by plaotec** [comment]
![3](https://github.com/Security-Onion-Solutions/securityonion/assets/63425694/bdd9d3eb-039e-442e-90ec-95fc024719ee)

**Comment by plaotec** [comment]
![1](https://github.com/Security-Onion-Solutions/securityonion/assets/63425694/addb2414-af8e-4e53-b6b0-98debcf3265a)

**Comment by plaotec** [comment]
![Screenshot 2023-08-29 123149](https://github.com/Security-Onion-Solutions/securityonion/assets/63425694/83e093e9-699e-423e-a205-e79b16e42a43)

**Comment by plaotec** [comment]
![Screenshot 2023-08-29 124407](https://github.com/Security-Onion-Solutions/securityonion/assets/63425694/09afb726-6744-4885-9178-84535e8ed11e)

**Comment by plaotec** [comment]
Did another v2.4 install from ISO, no install error at the end time but still not getting Suricata Alerts...

Please let me know HOW to setup the homenet permanently  .. tried editing the /opt/so/conf/suricata/suricata.yaml file, to change the homenet, but after change, and restarted so-suricata, the homenet went back to the default values.

**Comment by dougburks** [comment, marked as answer]
> Please let me know HOW to setup the homenet permanently .. tried editing the /opt/so/conf/suricata/suricata.yaml file, to change the homenet, but after change, and restarted so-suricata, the homenet went back to the default values.

Please see the Suricata section of our documentation:
https://docs.securityonion.net/en/2.4/suricata.html#home-net

If you still don't see Suricata alerts, please see the Troubleshooting Alerts section of our documentation:
https://docs.securityonion.net/en/2.4/suricata.html#troubleshooting-alerts
